examples/README.md

```diff
@@ -1,29 +1,19 @@
-# Simple Examples
+# LlamaIndexTS Examples
 
-Before running any of the examples, make sure to set your OpenAI environment variable:
+Before running any of the code examples,
+make sure you have basic knowledge of the [LlamaIndexTS](https://ts.llamaindex.ai/).
 
-```bash
-export OPENAI_API_KEY="sk-..."
-```
-
-There are two ways to run the examples, using the latest published version of `llamaindex` or using a local build.
-
-## Using the latest published version
+## Usage
 
-Make sure to call `npm install` before running these examples:
+```shell
+# export your API key
+export OPENAI_API_KEY="sk-..."
 
-```bash
-npm install
+npx ts-node ./chatEngine.ts
 ```
 
-Then run the examples with `ts-node`, for example `npx ts-node vectorIndex.ts`
-
-## Using the local build
```

Comment:
@himself65 why did you remove how to use the local build?

Reply:
No you don’t need that if you’re using pnpm. It will automatically use the monorepo I believe

Reply:
https://pnpm.io/workspaces#workspace-protocol-workspace

```
By default, pnpm will link packages from the workspace if the available packages match the declared ranges. For instance, foo@1.0.0 is linked into bar if bar has "foo": "^1.0.0" in its dependencies and foo@1.0.0 is in the workspace. However, if bar has "foo": "2.0.0" in dependencies and foo@2.0.0 is not in the workspace, foo@2.0.0 will be installed from the registry. This behavior introduces some uncertainty.
```

Reply:
In my tests this only worked if you set `"llamaindex": "workspace:*"`, but I set it on purpose to `"llamaindex": "latest"` so it works without pnpm (for our users).